start-pulsar.sh

```
@@ -1,11 +1,7 @@
-#!/bin/bash
-set -e
+bin/pulsar standalone &
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)